### PR TITLE
⭐️ add example with Chef Workstation run

### PR DESCRIPTION
**Pull request by chris-rock**

### CODE REVIEW

File: .github/actions/spelling/expect.txt [hunk 1]
```diff
@@ -1,20 +1,27 @@
...
 Linux
+metadata
 mondoo
 mondoohq
 Mondoo
+mkdir
 opensource
 orgs
 pem
+Policyfile
```
Comment (tas50) [inline .github/actions/spelling/expect.txt:17]: This should go in our main dictionary

File: .github/actions/spelling/expect.txt [hunk 2]
```diff
@@ -1,20 +1,27 @@
...
 SLES
+ssh
```
Comment (tas50) [inline .github/actions/spelling/expect.txt:19]: This should go in our main dictionary

File: .github/actions/spelling/expect.txt [hunk 3]
```diff
@@ -1,20 +1,27 @@
...
 github
 http
 https
 img
 Linux
+metadata
 mondoo
 mondoohq
 Mondoo
+mkdir
```
Comment (tas50) [inline .github/actions/spelling/expect.txt:13]: This should go in our main dictionary

File: README.md [hunk 4]
```diff
@@ -32,6 +32,10 @@ It does:
 | -------------- | ------------- | -----------------------------------|
 | `default['mondoo']['registration_token']` | `change_me` | Mondoo Registration Token that is used to retrieve client credentials
 
+## Run Mondoo Cookbook with Chef Run
```
Comment (tas50) [inline README.md]:
```suggestion
## Run Mondoo Cookbook with chef-run
```

File: examples/README.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
+# Run Mondoo Cookbook with Chef Run
```
Comment (tas50) [inline examples/README.md]:
```suggestion
# Run Mondoo Cookbook with chef-run
```

File: .github/actions/spelling/expect.txt (hunk 1 above)
Comment (chris-rock) [reply .github/actions/spelling/expect.txt:17]: Can we add the spell-check as a separate PR?

Comment (tas50) [reply .github/actions/spelling/expect.txt:17]: Yep

**Pull request merged**